packages/ui-kit/src/components/rss-feeds.js

```diff
@@ -1,23 +1,50 @@
 import React from 'react';
 import useSWR from 'swr';
-import Parser from 'rss-parser';
 import PropTypes from 'prop-types';
 import LoadingSpinner from '@commercetools-uikit/loading-spinner';
 import ContentNotifications from './content-notifications';
 import RssFeedTable from './rss-feed-table';
 
+const parseFieldValue = (element) =>
+  element.innerHTML.replace('<![CDATA[', '').replace(']]>', '');
```

Comment:
```suggestion
[ ...element.childNodes ].map(textOrCdataNode => textOrCdataNode.data).join()
```
(beauty contest in sticking to DOM apis  ;-)